tools/vsimporter/xib2nib/NIBWriter.cpp

```diff
@@ -302,21 +302,47 @@ XIBObject *NIBWriter::GetProxyFor(XIBObject *obj)
 }
 
 std::map<std::string, std::string> _g_exportedControllers;
+std::map<std::string, std::string> _g_exportedControllersbyStoryboardId;
```

Comment:
It looks like on reference platform there's no plist entry to describe storyboard ids, it just goes by the UIViewControllerIdentifierToNibNames key.